source/_integrations/broadlink.markdown

```diff
@@ -39,25 +39,28 @@ The entities are divided into three subdomains:
 
 ## Remote
 
-The `remote` entities allow you to control Broadlink universal remotes. You can learn IR codes, send the codes you have learned and send RF codes (use the base64 functionality). These entities are created automatically when you configure a device that has IR/RF capabilities.
+The `remote` entities are created automatically when you configure a universal remote. Use them to learn, store and send IR and RF commands.
 
-### Learning IR codes
+### Learning IR/RF codes
 
-Use the `remote.learn_command` service to learn IR codes.
+Use the `remote.learn_command` service to learn IR/RF codes.
 
 | Service data attribute | Optional | Description                          |
 | ---------------------- | -------- | ------------------------------------ |
 | `entity_id`            | no       | ID of the remote.                    |
 | `device`               | no       | Name of the device to be controlled. |
 | `command`              | no       | Names of the commands to be learned. |
+| `command_type`         | yes      | Type of the commands to be learned.  |
 | `alternative`          | yes      | Are they toggle commands?            |
 
-Example 1: Learn a single command
+Example 1: Learn IR command
+
+Learning IR commands is easy. Just call `remote.learn_command`, point the remote at the Broadlink device and press the button when the LED blinks.
```

Comment:
"is easy", "just"

It might not be as easy for everybody and not something anybody "just" can do. This might be experienced in a condescending way for those people. Try to avoid using those terms.